CHANGELOG.md

```diff
@@ -4,6 +4,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.10.0
+
+### Added
+*  Recommendation in `getting-started.md` that OneFuzz users include a `.onefuzz` file in their project root directory for security tool detection [#2236](https://github.com/microsoft/onefuzz/pull/2236)
+* Agent:  New `libfuzzer_dotnet_fuzz` task [#2221](https://github.com/microsoft/onefuzz/pull/2221)
+* CLI: Allow dotnet endpoint and functions configuration in the CLI [#2190](https://github.com/microsoft/onefuzz/pull/2190)
```

Comment:
```suggestion
```
My suggestion is to remove this because it's only here for our testing and not for end user consumption.